notebooks/Deep IV Examples.ipynb

```diff
@@ -0,0 +1,718 @@
+    "                                   keras.layers.Dropout(0.17),\n",
+    "                                   keras.layers.Dense(64, activation='relu'),\n",
+    "                                   keras.layers.Dropout(0.17),\n",
+    "                                   keras.layers.Dense(32, activation='relu'),\n",
+    "                                   keras.layers.Dropout(0.17),\n",
+    "                                   keras.layers.Dense(1)])"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we'll instantiate the `DeepIVEstimator` class using these models.  Defining the response model *outside* of the lambda passed into constructor is important, because (depending on the settings for the loss) it can be used multiple times in the second stage and we want the same weights to be used every time."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "deepIvEst = DeepIVEstimator(n_components = 10, # number of gaussians in our mixture density network\n",
+    "                            m = lambda z, x : treatment_model(keras.layers.concatenate([z,x])), # treatment model\n",
+    "                            h = lambda t, x : response_model(keras.layers.concatenate([t,x])),  # response model\n",
+    "                            n_samples = 1, # number of samples to use to estimate the response\n",
+    "                            use_upper_bound_loss = False, # whether to use an approximation to the true loss\n",
+    "                            n_gradient_samples = 1, # number of samples to use in second estimate of the response (to make loss estimate unbiased)\n",
+    "                            optimizer='adam', # Keras optimizer to use for training - see https://keras.io/optimizers/ \n",
+    "                            s1=100, # number of epochs to train treatment model\n",
+    "                            s2=100) # number of epochs to train response model"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Fitting and predicting using the model\n",
+    "Now we can fit our model to the data:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "c:\\users\\kebatt\\source\\repos\\projectalicecore\\econml\\deepiv.py:327: UserWarning: Output \"model_2\" missing from loss dictionary. We assume this was done on purpose, and we will not be expecting any data to be passed to \"model_2\" during training.\n",
+      "  model.compile(self._optimizer)\n"
+     ]
+    },
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Epoch 1/100\n",
+      "5000/5000 [==============================] - 1s 244us/step - loss: 1.8314\n",
```

Comment:
Is there a way to suppress these? They make it hard to scroll through the notebook.

Reply:
Created issue #35 to address this comment later.